tools/nd_freeze

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+#
+# Install access to the NeuroDebian snapshot repository that allows access
+# to old packages based on dates and version numbers.
+#
+# The date/time format sent to NeuroDebian to get the repo info is in the
+# format yyyymmddThhmmssz or yyymmdd. However, the script will handle and
+# reformat a number of date/time formats. Quotes are required if there is a
+# space between the date and time components:
+#
+#   yyyymmddThh:mm:ssZ
+#   yyyymmdd
+#   mm/dd/yyyy
+#   yyyy-mm-dd
+#   yyyymmddThh:mm
+#   "yyyy-mm-dd hh:mm:ss"
+#   "mm/dd/yy hh:mm:ss"
```

Comment:
are the times in the last types of entry in UTC or local zone?  I guess in the topmost it is the UTC?

Reply:
The timezone of importance is the one at snapshot-neuro.debian.net. I assume it is UTC so the user should be notified that the date/time they enter is not translated into UTC.